lib/css/components/post-summary.less

```diff
@@ -186,6 +195,7 @@
             }
         }
 
+        // TODO: Remove and replace with references with `s-badge s-badge__bounty`
```

Comment:
nit

```suggestion
        // TODO DEPRECATED: Remove and replace with references with `s-badge s-badge__bounty`
```

Reply:
Do the same with all below - I'm being lazy and not marking all with a code suggestion.